plugins/source/tailscale/client/testing.go

```diff
@@ -23,31 +25,26 @@ func MockTestHelper(t *testing.T, table *schema.Table, createService func(*httpr
 	mux := httprouter.New()
 	ts := httptest.NewUnstartedServer(mux)
 	defer ts.Close()
+	if err := createService(mux); err != nil {
+		t.Fatalf("failed to createService: %v", err)
+	}
+	var acl tailscale.ACL
+	if err := faker.FakeObject(&acl); err != nil {
+		t.Fatalf("failed to fake ACL: %v", err)
+	}
 
-	newTestExecutionClient := func(_ context.Context, logger zerolog.Logger, _ specs.Source, _ source.Options) (schema.ClientMeta, error) {
-		err := createService(mux)
+	mux.GET("/api/v2/tailnet/:tailnet/acl", func(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
```

Comment:
Should this code be under `createAcls` in `plugins/source/tailscale/resources/services/acl/acls_mock_test.go`?

Reply:
yes, will move. some deprecated code